doc/content/getting-started/ttn/addresses.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Addresses"
+description: ""
+---
+
+The addresses where you can reach The Things Network network consist of the cluster ID and `cloud.thethings.network`.
+
+You can update the examples below with your cluster ID by filling it here.
+
+{{< cluster-selector >}}
+
+## API Endpoints
+
+The Application Server, Join Server, and Network Server APIs are all available in your regional cluster:
+
+<p>
+<code data-content="cluster-address">
+<span data-content="cluster-id"></span>.cloud.thethings.network
+</code>
+</p>
+
+However, the Identity Server APIs are only available in the `eu1` cluster:
+
+<p>
+<code data-content="cluster-address">
+eu1.cloud.thethings.network
+</code>
+</p>
+
+{{< warning >}}End-Device Registry API requests or other Identity Server API requests to any cluster other than `eu1` will fail. {{</ warning >}}
+
+## Account and OAuth
+
+User accounts are stored in the `eu1` cluster. For registering users, logging in, changing your password and for performing OAuth flows, you need to use the following address:
+
+<p>
+<code data-content="cluster-address">
+https://eu1.cloud.thethings.network/oauth
+</code>
+</p>
+
+## Console
```

Comment:
I reordered this as part of a PR merged yesterday. Please use the same order. Console is going up.